### PR TITLE
Implemented support for multiple file types for getFrameAtTime

### PR DESCRIPTION
**Pull request by theMadness**
Some other generic cleanup.

### CODE REVIEW

**Comment by theMadness** [comment]
cfr #32

File: adapter/ffmpeg_frame.php [hunk 1]
```diff
@@ -34,11 +34,11 @@ public function getPresentationTimestamp() {
     }
     
     public function resize($width, $height, $cropTop = 0, $cropBottom = 0, $cropLeft = 0, $cropRight = 0) {        
-        return $this->adaptee->resize($width, $height, $cropTop, $cropBottom, $cropLeft, $cropRight);
+        $this->adaptee->resize($width, $height, $cropTop, $cropBottom, $cropLeft, $cropRight);
```
Comment (char0n) [inline adapter/ffmpeg_frame.php]: why remuving the returns ?

Comment (theMadness) [inline adapter/ffmpeg_frame.php]: FFmpegFrame::resize is a void function, we're returning the result of a void function.

Comment (char0n) [inline adapter/ffmpeg_frame.php]: Thanks for explanation. Merging.

**Comment by char0n** [comment]
Anyway before creating the release I would like to verify the tests first.

**Comment by theMadness** [comment]
Legitimate.

**Comment by char0n** [comment]
Can you advice how to install ffmpeg on linux machine nowadays ?

**Comment by theMadness** [comment]
most package managers can do it pretty easily, just google the distro name and install ffmpeg and it will likely return some form of "apt-get install ffmpeg" or "yum install ffmpeg" or something like that.

**Pull request merged**